bin/salmon_summarizedexperiment.r

```diff
@@ -22,7 +22,9 @@ if (info$size == 0){
 }
 
 counts = read.csv(counts_fn, row.names=1, sep="\t")
+counts = counts[,2:ncol(counts)] # remove gene_name column
 tpm = read.csv(tpm_fn, row.names=1, sep="\t")
+tpm = tpm[,2:ncol(tpm)] # remove gene_name column
```

Comment:
in the final `tsv` files, `gene_name` is the second column. Wouldn't this rather remove the `gene_id` column?

Reply:
I thought about that, but there are still people who would prefer to have the ugly ID instead. I tried to be the most inclusive as possible. Would this be very bad for you?

Reply:
No, I'm absolutely in favor of keeping the ID as primary identifier and as long as possible. 

I just thought that this line *actually* removes the gene_id, instead of removing the gene_name as mentioned in the comment.

Reply:
oh, sorry. No, that is to get only the numeric data to create the R object. Both IDs will be in the rowData of that object. The TSV files should have both IDs. Sorry I misunderstood.

Reply:
In that script, when reading the TSV, the first column goes to row.names, and the second column gets to be the first. So I need to remove it to get only the count data. I hope this is better explained now. I am having a silly day today and I am slow. Let me know.